crates/db/store-sled/src/asm/mmr.rs

```diff
@@ -0,0 +1,507 @@
+//! Sled-backed MMR Database implementation
+
+use strata_db_types::{
+    DbError, DbResult,
+    mmr_helpers::{
+        get_peaks, leaf_index_to_mmr_size, leaf_index_to_pos, pos_height_in_tree, sibling_pos,
+    },
+    traits::MmrDatabase,
+};
+use strata_merkle::{CompactMmr64B32 as CompactMmr64, Sha256Hasher, hasher::MerkleHasher};
+use typed_sled::SledTree;
+
+use super::schemas::{AsmMmrMetaSchema, AsmMmrNodeSchema, MmrMetadata};
+
+// Hash type for 32-byte hashes
+type Hash = [u8; 32];
```

Comment:
This *might* be able to be switched to `Buf32` like we've been using elsewhere, not sure though.